p2p/client.go

```diff
@@ -187,7 +190,11 @@ func (c *Client) ConnectionGater() *conngater.BasicConnectionGater {
 
 // Info returns client ID, ListenAddr, and Network info
 func (c *Client) Info() (p2p.ID, string, string) {
-	return p2p.ID(c.host.ID().String()), c.conf.ListenAddress, c.chainID
+	rawKey, err := c.privKey.GetPublic().Raw()
+	if err != nil {
+		c.logger.Error("failed to extract raw bytes from p2p key", "error", err)
```

Comment:
Why wouldn't we return here?

Reply:
are you right, we shouldn't consume the error. fixed.